### DISCUSSION TITLE
⭐ Submit your application to our showcase section

### DISCUSSION THREAD

**Discussion by marekhozak10** [clientIO/joint#2721] [Your stories]
👋 **Built an awesome diagramming application and want to share it with the world?** Submit your application to our showcase section by providing the details below. 

Please note that we carefully select projects for our page, so we can't guarantee that every submission will be featured.

### Submit your application
Comment to this thread and share a few basic information:
- Name of your company (+ product if applicable)
- Website of your diagramming application (or company website if you don't have a dedicated page to the JointJS-powered feature)
- Segment (for instance Marketing automation)
- 3-word description of your application
- Short description of your company and product (max 400 characters)
- Up to 5 images (16:9) of the diagramming application

**Alternatively, you can just share the URL of your application and we will figure the details out ourselves.** 🤝

This is what the showcase of your awesome application will look like once it's live:

<img width="710" alt="Screenshot 2024-07-26 at 15 37 33" src="https://github.com/user-attachments/assets/fda6ff3a-eece-499e-b755-d5d375b6ce9d">

We can't wait to see what you've created with our diagramming library! 🤩

**Comment by DeepJam** [comment]
Some basic details as requested
Company name is ParaQualis LLC
SuccessQMS.ai
The segment is the Pharmaceutical Quality Operations,
QMS Compliance Management

ParaQualis has been set up by experts in the management of quality control documents (for instance Standard Operating Procedures or SOPs) to reduce the load on the periodic review of SOPs, make the review of SOPs during M&A activity and the writing of SOPs in response to regulatory change much more manageable.   The first product is a tool that manages the compliance checking of SOPs using a suite of agents that work with you and check each others work to ensure your documents are compliant (PQ-Compliance).  There is also a component which manages the development of new SOPs (PQ-SOP) and a lifecycle management tool (PQ-QMS).  Each of the product combines agents filling different roles in the quality management process to give you a complete view of the SOP covering regulations in multiple countries.  PQ-SOP uses the networking tool to provide insight into the structure of SOPs and to show the SOP model as both a process flow and a knowledge graph.  PQ-QMS models the structure of the QMS, integrating the different SOPs, supporting documents and company policies into a single model that can be browsed to show the connections between components, improving reuse and compliance

Screen Dumps will follow in a couple of days, not all finished yet

**Reply by marekhozak10** [reply]
Craig, thank you for the submission! :) To include it on our website, we would need a screenshot – once you share it, we're happy to review it and include it in our showcase section.

**Reply by DeepJam** [reply]
<img width="2154" height="1780" alt="Screenshot from 2026-06-23 07-41-22" src="https://github.com/user-attachments/assets/e9ac62de-42be-4cd5-b0a7-1cc618b27fc6" />

Hope the discount is still available, this is the image of a word document being converted to a process flow and using Jointjs to render it

**Comment by DeepJam** [comment]
Done

**Comment by bjgross10767** [comment]
Company (+ product): Macula Systems, LLC — Macula Blaze
Website: https://www.maculasys.com/blaze-mdp
Segment: Enterprise Data and AI (data governance and ontology modeling)
3-word description: AI Context Builder
Short description:

Macula Blaze is an enterprise data-context platform that sits above modern cloud platforms like Microsoft Fabric, Databricks, and Snowflake. Its visual canvas lets teams model ontologies, bind entities to physical data, and apply governance rules without code. Built by Macula Systems, a Data and AI firm, Blaze turns fragmented data estates into a trustworthy, AI-ready business context layer.

Development is just starting, so will update with pictures once we have them.
